client/src/App.js

```diff
@@ -70,23 +75,57 @@ function App() {
     setShowAddModal(false)
   }
 
-  const onSuccessAddSubscriber = () => {
+  const onSuccessAddSubscriber = (success) => {
     setShowAddModal(false)
+    refreshSubscribers()
+    setAlertMessage(success)
+    setAlertVariant('success')
+    window.setTimeout(() => {
+      setAlertMessage('')
+      setAlertVariant('')
+    }, 5000)
+  }
+
+  const onFailureAddSubscriber = (error) => {
```

Comment:
This function shows an alert with an error message when an error occurs adding a subscriber